src/components/NavBar/NavbarLinks.jsx

```diff
@@ -35,6 +35,8 @@ const NavbarLinks = () => {
     { label: 'Messages', path: '/PASS/messages' }
   ];
 
+  const tabRoutes = ['clients', 'documents', 'messages'];
```

Comment:
This duplicates the data found in routesArray immediately above. Why can't you just use the routes array?

Reply:
I think I could do that, I thought this was what was being described as the suggested solution originally. But this can be adjusted, I'm going to include location name inside routesArray.